usage-guides/historical-events.md

```diff
@@ -47,6 +47,18 @@ Events can be exported as CSV, click on `Export CSV` button and accept prompt to
 
 By default, all events will be processed in accounting, but you can ignore unwanted events, so they won't be processed. You can click on the three dots to display the options for the group of events, and click `Ignore events in accounting`/`Unignore events in accounting`.
 
+## Refreshing Events
+
+You can choose to refresh all events by clicking the main `Refresh` button, or you can open the menu and choose to refresh an individual type of event or events on a certain account.
+
+![Refreshing events](/images/refreshing_events.png)
+
+To see the query status, you can click the button here:
+
+![See query status](/images/see_query_status.png)
+
+![Events query status](/images/events_query_status.png)
```

Comment:
can you also mention the events decodign step and its query status check?

Reply:
Yes this step, they are both related

Reply:
No. It's not about telling the user how to do it manually, but letting them know what the next step is and what does rotki do during this step. This is not clear.

Reply:
Do i miss anything? https://github.com/rotki/docs/blob/6b2cefb5e584ab57246fdd4a76d3a48d8b01d5e3/usage-guides/historical-events.md?plain=1#L60